Looped/Interface/Create/CreateViewController.swift

```diff
@@ -4,12 +4,68 @@ protocol CreateViewControllerFlowDelegate: class, DetailsShowable {}
 
 final class CreateViewController: ViewController {
     
+    // MARK: - Actions
+    @objc func segmentedTapped(sender: UISegmentedControl) {
+        changeContainerViewController(to: sender.selectedSegmentIndex)
+    }
+    
     // MARK: - Properties
     private let createView = CreateView()
-    weak var flowDelegate: CreateViewControllerFlowDelegate?
+    var flowDelegate: CreateViewControllerFlowDelegate?
+    let createOnlineViewController = CreateOnlineViewController()
+    let createLibraryViewController = CreateLibraryViewController()
     
     // MARK: - Overrides
     override func loadView() {
         view = createView
     }
+    
+    override func setupNavigationItem() {
+        super.setupNavigationItem()
+        title = "Create GIF"
+    }
+    
+    override func setupProperties() {
+        super.setupProperties()
+        setupSegmentedControl()
+        changeContainerViewController(to: 0)
+    }
+    
+    // MARK: - Private
+    private func setupSegmentedControl() {
+        createView.segmentedControl.setTitle("URL", forSegmentAt: 0)
+        createView.segmentedControl.setTitle("Library", forSegmentAt: 1)
+        createView.segmentedControl.addTarget(self, action: #selector(segmentedTapped), for: .valueChanged)
+    }
+    
+    private func changeContainerViewController(to index: Int) {
+        switch index {
+        case 0: // Create from URL
+            removeContainer(viewController: createOnlineViewController)
+            
+            addChild(createOnlineViewController)
```

Comment:
kod tutaj sie powtarza. Może dobrze jest to ubrać w funkcję albo extension do viewControllera?